config/Info.plist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>English</string>
+	<key>CFBundleDisplayName</key>
+	<string>X-Moto</string>
+	<key>CFBundleName</key>
+	<string>X-Moto</string>
+	<key>CFBundleExecutable</key>
+	<string>helics</string>
+	<key>CFBundleIconFile</key>
+	<string>helics.icns</string>
+	<key>CFBundleIdentifier</key>
+	<string>lib.gmlc-tdc.helics</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>1.0</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>%VERS%</string>
+	<key>CFBundleVersion</key>
+	<string>%VERS%</string>
+	<key>NSHumanReadableCopyright</key>
+	<string>© 2017 HELICS</string>
+	<key>NSPrincipalClass</key>
+	<string>NSApplication</string>
+</dict>
+</plist>
```

Comment:
Is there anything in this file that is likely to change over time (other than the copyright once a year)